internal/policy/policy_test.go

```diff
@@ -57,8 +57,10 @@ func TestInitializeNamespace(t *testing.T) {
 			t.Fatal(err)
 		}
 
-		err = InitializeNamespace(repo)
-		assert.ErrorIs(t, err, ErrPolicyExists)
+		// Check if policy with zero hash is treated as uninitialized
+		if err := InitializeNamespace(repo); err != nil {
+			t.Fatal(err)
+		}
```

Comment:
I see you've done this for the RSL ref, but can you add a subsequent check here where the policy ref is set to something else? Note that it doesn't **need** to be valid policy, the ref just needs to be non zero. Even setting it to `gitinterface.EmptyBlob()` will suffice.